doc/memos/rdm0.md

```diff
@@ -0,0 +1,146 @@
+- RIOT Maintainers Community (RIOT-MC)  
+- Developer Memo: 0  
```

Comment:
@miri64 I don't see how the number here is more or less confusing than just having it in the file name? Either we have RDM numbers (in this case we have to modify/assign at time of merge) or we do not have numbers and then this aspect disappears altogether?